lib/notifier.js

```diff
@@ -298,11 +300,18 @@ function addItem(item, callback) {
       if (err) {
         return callback(err);
       }
-      apiClient.postItem(data, function (err, resp) {
-        if (typeof callback === 'function') {
-          callback(err, data, resp);
+      if(SETTINGS.enabled){
```

Comment:
Calling buildItemData() even when not actually sending the error has the advantage that all the code parsing and building the error is still run. This way potential errors in Rollbar occur while developing and not only when the app is run in production with Rollbar enabled.

Of course you could argue that setting enabled = false should disable *as much* code as possible in Rollbar, not as little.

Should I still move the check before buildItemData().